layouts/partials/navbar.html

```diff
@@ -1,7 +1,7 @@
 {{ $cover := .HasShortcode "blocks/cover" }}
 <nav class="js-navbar-scroll navbar navbar-expand navbar-dark {{ if $cover}} td-navbar-cover {{ end }}flex-column flex-md-row td-navbar">
 	<a class="navbar-brand" href="{{ .Site.Home.RelPermalink }}">
-		<span class="navbar-logo">{{ with resources.Get "icons/logo.svg" }}{{ ( . | minify).Content | safeHTML }} {{ end }}</span><span class="text-uppercase font-weight-bold">{{ .Site.Title }}</span>
+		<span class="navbar-logo">{{ if (fileExists "icons/logo.svg") }}{{ with resources.Get "icons/logo.svg" }}{{ ( . | minify).Content | safeHTML }}{{ end }}{{ end }}</span><span class="text-uppercase font-weight-bold">{{ .Site.Title }}</span>
```

Comment:
This doesn't work for me and returns false even if the file exists. I had tried this before and I couldn't get fileExists to work with files in assets/

Reply:
Huh, let me play around with it some more, it worked for me with one example....

Reply:
Aha, figured it out. You have to actually specify the assets/ directory as well for fileExists - you don’t pass fileExists the path to the file as it appears with hugo server but use the path on the disk.